### PR TITLE
📖 docs: add small blurb to README describing MyST

### PR DESCRIPTION
**Pull request by agoose77**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ pip install jupyterlab_myst
 
 ## Usage
 
+[MyST}] is a flavour of Markdown, which combines the fluid experience of writing Markdown with the programmable extensibility of reStructuredText. This extension for JupyterLab makes it easier to develop rich, computational narratives, technical documentation, and open scientific communication.
```
Comment (rowanc1) [inline README.md:27]: Is the punctuation intentional? If we rebase, we can make this a link easily with your other changes.

```suggestion
MyST is a flavour of Markdown, which combines the fluid experience of writing Markdown with the programmable extensibility of reStructuredText. This extension for JupyterLab makes it easier to develop rich, computational narratives, technical documentation, and open scientific communication.
```

Comment (agoose77) [reply README.md:27]: A very generous way of saying wtf is this ;) It was a link at one point, but something in my text buffer seems to have been moved around.

Comment (agoose77) [reply README.md:27]:
```suggestion
[MyST](https://myst-tools.org/docs/mystjs/quickstart-myst-markdown) is a flavour of Markdown, which combines the fluid experience of writing Markdown with the programmable extensibility of reStructuredText. This extension for JupyterLab makes it easier to develop rich, computational narratives, technical documentation, and open scientific communication.
```

Comment (rowanc1) [reply README.md]: Was wondering if it was a part of the logo or something!

**Pull request merged**